src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -229,7 +229,10 @@ namespace DataHVACGlobals {
 
     extern int const CoilVRF_FluidTCtrl_Cooling;
     extern int const CoilVRF_FluidTCtrl_Heating;
-    extern int const CoilDX_SubcoolReheat;
+
+    extern int const coilNormalMode;        // Normal operation mode
+    extern int const coilEnhancedMode;      // Enhanced operation mode
+    extern int const coilSubcoolReheatMode; // SubcoolReheat operation mode
```

Comment:
@Myoldmopar These are moved here from CoilCoolingDX.hh. Are you ok with that? 
Would we prefer an `enum class` here?

Reply:
An enum class is always preferred when it is possible, but it doesn't _necessarily_ be fixed as a part of this.